### DISCUSSION TITLE
Can't log in to Vercel with GitHub auth, error occurs [This user account is blocked]

### DISCUSSION THREAD

**Discussion by rexodim** [community/community#63250] [New to GitHub]
Can't log in to Vercel with GitHub auth, error occurs [This user account is blocked]

**Comment by w8ste** [comment]
@rexodim Try sending an email to [registration@vercel.com](mailto:registration@vercel.com) and explain your situation. This is their support service for these cases. Your account is probably blocked because your account is rather new.

Hope this helps :)
